style.css

```diff
@@ -80,6 +80,130 @@ h1 {
   color: #fff;
 }
 
+.toggle-list-top li,
+.toggle-list-bottom li {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: bolder;
+  font-size: 1rem;
+  color: #949eff;
+  border: 1px solid #dadbff;
+  border-radius: 0.5rem;
+  padding: 0 0.3rem;
+  margin: 1rem 0.3rem 0 0;
+  background-color: #ebebff;
+}
+
+.modal {
+  opacity: 1;
+  display: none;
+  position: fixed;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgba(0, 0, 0, 0.8);
+}
+
+.modal-content {
+  background-color: #fefefe;
+  margin: 2rem auto;
+  padding: 20px;
+  border: 1px solid #888;
+  height: auto;
+  width: 65%;
+}
```

Comment:
- [ ] The modal popup does not display correctly on mobile devices 📱 as specified by the Figma design
You could use media queries to target this `.modal-content` at `max-width` of `768px` and increase its width. Maybe 95% would be better?

- [ ] The paragraph text is too narrow. Please adjust its width as well 🙏 
- [ ] Generally make it look as close as possible to the Figma design

### Your Design
![image](https://user-images.githubusercontent.com/46839548/162578939-57b3ffef-c2e5-4923-b943-243dc17331f6.png)

### Figma Design
![image](https://user-images.githubusercontent.com/46839548/162578987-8bbf7f4c-72dd-4fd1-a198-16563ce3ecb8.png)